### ISSUE TITLE
v0.5 - Plugin boilerplate is out of date

### ISSUE THREAD

**Issue by jordanmack** [serverless/serverless#835]
`sls plugin create` installs a plugin boilerplate which is out of date.

```
TypeError: Cannot read property 'addAction' of undefined
    at PluginBoilerplate.registerActions (/home/username/Desktop/sls-test/plugins/testing/index.js:61:13)
```

**Comment by neaghfoz** [comment]
Push some fixes in #837 , waiting for merge

**Comment by ac360** [comment]
Thanks @jordanmack and @neaghfoz :)  Merged.
